src/Uno.UI.RuntimeTests/MUX/Microsoft_UI_Xaml_Controls/TimePicker/TimePickerIntegrationTests.cs

```diff
@@ -284,6 +284,8 @@ public async Task SelectingTimeSetsSelectedTime()
 		await DateTimePickerHelper.OpenDateTimePicker(timePicker);
 		await TestServices.WindowHelper.WaitForIdle();
 
+		// Uno Specific: the test will fail on 24-hour systems since it expects a period LoopingSelector
+		timePicker.ClockIdentifier = ClockIdentifiers.TwelveHour;
```

Comment:
`SetupTimePickerTestAsync` is used by a lot of tests. This workaround is only needed for `SelectingTimeSetsSelectedTime`, so it would be unnecessary to modify a path that most tests go through

Reply:
IMO, It's better to force it for all tests to avoid having this kind of problem in future.